src/coreclr/interop/referencetrackertypes.hpp

```diff
@@ -43,6 +46,9 @@ class DECLSPEC_UUID("04b3486c-4687-4229-8d14-505ab584dd88") IFindReferenceTarget
     STDMETHOD(FoundTrackerTarget)(_In_ IReferenceTrackerTarget* target) = 0;
 };
 
+//11d3b13a-180e-4789-a8be-7712882893e6
+const GUID IID_IReferenceTracker = { 0x11d3b13a, 0x180e, 0x4789, { 0xa8, 0xbe, 0x77, 0x12, 0x88, 0x28, 0x93, 0x36} };
```

Comment:
```suggestion
const GUID IID_IReferenceTracker = { 0x11d3b13a, 0x180e, 0x4789, { 0xa8, 0xbe, 0x77, 0x12, 0x88, 0x28, 0x93, 0xe6} };
```

Reply:
I believe this is what is causing a bunch of the test failures.

Reply:
Ok

Reply:
After the fix, all checks have passed. I'm sorry for the typo